### DISCUSSION TITLE
Is there any way to use swayosd or something similar in niri?

### DISCUSSION THREAD

**Discussion by NotMugil** [YaLTeR/niri#444] [Q&A] [answered]
I tried replacing  ` XF86AudioRaiseVolume allow-when-locked=true { spawn "wpctl" "set-volume" "@DEFAULT_AUDIO_SINK@" "0.1+"; } ` with `  XF86AudioRaiseVolume allow-when-locked=true { "bash" "-c" "swayosd-client --output-volume raise" } ` . It didn't work.

Edit: Created a bash script  and used XF86AudioRaiseVolume { spawn "bash" "-c" "exec ~/.scripts/swayosd.sh raise"

**Comment by ocharles** [comment, marked as answer]
swayosd works fine for me with:

```
    XF86AudioLowerVolume { spawn "swayosd-client" "--output-volume" "lower"; }
    XF86AudioMute { spawn "swayosd-client" "--output-volume" "mute-toggle"; }
    XF86AudioRaiseVolume { spawn "swayosd-client" "--output-volume" "raise"; }
    XF86MonBrightnessDown { spawn "swayosd-client" "--brightness" "lower"; }
    XF86MonBrightnessUp { spawn "swayosd-client" "--brightness" "raise"; }
```
